packages/components/src/Stack/Stack.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { system } from '@marigold/system';
+import { Box } from '../Box';
+import flattenChildren from 'react-flatten-children';
```

Comment:
what about the other package that has more downloads? :)

Reply:
I would like to use React.Children.map then we need no extra dependency, ok?

Reply:
That will not cover the case of someone uses Fragments

Reply:
OK

Reply:
But then this test tests if flattenChildren working? 
test('supports two children' ...)

Reply:
It needs more nesting, not multiple Childs

Reply:
Please have one more look at my changes :) :D